CONTRIBUTING.md

```diff
@@ -0,0 +1,179 @@
+documentation.
+
+### Why document?
+
+Substrate is a new framework for building blockchains. Effective documentation allows new people to join and grow the project by having all necessary information.
+
+### Who is the audience?
+
+The documentation described here is intended to address a technical audience, i.e. those who expect
+to implement or exercise APIs or understand the internal dynamics of the blockchain system. These
+standards are not intended for end-user product documentation.
+
+The knowledge base houses instructions, conceptual overviews, and procedural documentation for 
+developers who are working on Substrate. This directory includes documentation on how to get 
+started, build, run, and test Substrate. You should organize the content that you create by 
+specific activities, such as testing, getting started, or by workflow topic.
+
+### What should I document?
+
+Document protocols, introduce essential concepts, and explain how everything fits together.
+
+- Conventions: e.g. this document about documentation, code style
+- System Design: e.g. network stack, consensus, runtime, assumptions
+- APIs: e.g. JSON-RPC, runtime functions
+- Protocols: e.g. schemas, encodings, configuration files
+- Tools: e.g. `subkey`, `build-spec`, `purge-chain`
+- Workflows: e.g. environment set up, test methodologies, where to find various
+  parts, how to get work done
+
+### What documentation style guidelines should I follow?
+
+It is important to follow documentation style guidelines to ensure that the documentation
+created by a large number of contributors can flow together. See
+[Documentation Style Guide](#documentation-style).
+
+### How can I link to other docs in my documentation?
+
+Use absolute paths starting with `/` and ending with `.md`, like `/overview/getting-started.md`.
+
+### How can I link to source code in my documentation?
+
+Use the rust-docs to link to specific parts of the Substrate source code, for example a module's `trait`. When referencing specific lines of code, copy those lines into the documentation.
+
+DO NOT link to specific lines of code -- Substrate changes often and links that reference specific locations in the code become inaccurate fast and are not easily maintainable.
+
+**Showing code can make an abstract idea concrete. It is now, however, a shortcut to writing a 
+clear explanation about how something works.**
+
+## Documentation Style
+
+It is important to create documentation that follows similar guidelines. This allows documentation
+to be clear and concise while allowing users to easily find necessary information.
+
+These are general style guidelines that can help create clearer documentation:
+
+- **Write in plain U.S. English.** You should write in plain U.S. English and try to avoid over-complicated words when you describe something.
+
+- **Avoid using pronouns such as "I" or "we".** These can be quite ambiguous when someone reads the
```

Comment:
```suggestion
- **Use you rather than "I" or "we".** These can be quite ambiguous when someone reads the
```

The explanation says something else than the bold part...

Reply:
The explanation says "you" is acceptable. Personally, I still try to steer clear but sometimes it really is the best way. Changed it to be more clear.